frontend/src/Container/RadioRedirector.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+class RadioRedirector extends React.Component {
+  static isHerokuDomain(hostname) {
+    return !!hostname.match(/herokuapp/);
```

Comment:
This line of code will return true for DEV, STAG and PROD. We only want to redirect from production with https://team-radio.herokuapp.com. Please change

Reply:
OK @dat-ng 

I forgot about DEV and STAG environment. I will take care of this problem.